config/initializers/sidekiq.rb

```diff
@@ -14,7 +14,14 @@
 Sidekiq::Extensions.enable_delay!
 
 Sidekiq.configure_server do |config|
-  config.redis = { url: Rails.application.secrets.redis_url, size: 9 }
+  config.redis = {
+    url: Rails.application.secrets.redis_url,
+    size: 9,
+    ssl_params: {
+      verify_mode: OpenSSL::SSL::VERIFY_NONE
+    }
```

Comment:
Couldn't we use the `ssl_params` secrets here?

Reply:
This is not something app-specific, hence doesn't need to be stored in secrets.